pkg/v3/flows/recovery.go

```diff
@@ -0,0 +1,63 @@
+package flows
+
+import (
+	"context"
+	"log"
+	"time"
+
+	ocr2keepers "github.com/smartcontractkit/ocr2keepers/pkg"
+	ocr2keepersv3 "github.com/smartcontractkit/ocr2keepers/pkg/v3"
+	"github.com/smartcontractkit/ocr2keepers/pkg/v3/postprocessors"
+	"github.com/smartcontractkit/ocr2keepers/pkg/v3/service"
+	"github.com/smartcontractkit/ocr2keepers/pkg/v3/tickers"
+)
+
+func newRecoveryProposalFlow(preprocessors []ocr2keepersv3.PreProcessor[ocr2keepers.UpkeepPayload], rs ResultStore, ms MetadataStore, rp RecoverableProvider, recoveryInterval time.Duration, logger *log.Logger, configFuncs ...tickers.ScheduleTickerConfigFunc) (service.Recoverable, Retryer) {
```

Comment:
I just noticed that the recovery proposal flow doesn't use the result store anymore. We will need to listen for result TTL events, but that can be another interface added later. You can remove the result store dependency.